packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -135,6 +135,10 @@ class HotRunner extends ResidentRunner {
 
     await refreshViews();
     for (FlutterDevice device in flutterDevices) {
+      // VM has to accept the kernel binary, there will be no reload report,
```

Comment:
Should the comment be VM has accepted the kernel binary?

Reply:
Changed the wording along the lines you suggested here and below.